src/interner.rs

```diff
@@ -254,6 +271,7 @@ impl<T: ?Sized> Interner<T> {
 
 /// Constructors to control the backing `HashSet`'s hash function
 impl<T: Eq + Hash + ?Sized, H: BuildHasher> Interner<T, H> {
+    #[cfg(not(feature = "static"))]
     /// Create an empty interner which will use the given hasher to hash the strings.
```

Comment:
> to hash the strings

Can you tell that this used to only be a string interner? More things I need to fix...

Reply:
I've gone through and tried to fix up the mentions of strings in f516a683df84ddd50369015dd9b933ccda8c50e1